templates/controller-cert-manager-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.controller.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "consul.fullname" . }}-webhook-cert-manager-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "consul.name" . }}
+    chart: {{ template "consul.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+data:
+  webhook-config.json: |-
+    [
+      {
+        "name": "{{ template "consul.fullname" . }}-controller-mutating-webhook-configuration",
+        "tlsAutoHosts": [
+          "{{ template "consul.fullname" . }}-controller-webhook",
+          "{{ template "consul.fullname" . }}-controller-webhook.{{ .Release.Namespace }}.svc",
+          "{{ template "consul.fullname" . }}-controller-webhook.{{ .Release.Namespace }}.svc.cluster.local"
+        ],
+        "secretName": "{{ template "consul.fullname" . }}-controller-webhook-cert",
+        "secretNamespace": "{{ .Release.Namespace }}"
+      }
+    ]
```

Comment:
Should we use YAML for the config file? It will be easier to template (vs hand-bombing json) and is more kube-native

Reply:
All of our prior art for configuration used JSON which is why I went down that route. I don't mind switching this to YAML but will that be a cause for confusion?

Reply:
You mean for Consul?

Reply:
For the server config and the client config.